yaml/stage.go

```diff
@@ -24,6 +24,7 @@ type (
 		Environment raw.StringSliceMap `yaml:"environment,omitempty" json:"environment,omitempty" jsonschema:"description=Provide environment variables injected into the container environment.\nReference: https://go-vela.github.io/docs/reference/yaml/stages/#the-environment-tag"`
 		Name        string             `yaml:"name,omitempty"        json:"name,omitempty"        jsonschema:"minLength=1,description=Unique identifier for the stage in the pipeline.\nReference: https://go-vela.github.io/docs/reference/yaml/stages/#the-name-tag"`
 		Needs       raw.StringSlice    `yaml:"needs,omitempty,flow"  json:"needs,omitempty"       jsonschema:"description=Stages that must complete before starting the current one.\nReference: https://go-vela.github.io/docs/reference/yaml/stages/#the-needs-tag"`
+		Continue    bool               `yaml:"continue,omitempty"    json:"continue,omitempty"    jsonschema:"description=Stage will continue executing if other stage fails if set to false; default is true"`
```

Comment:
The PR description talks about setting it to `true`, but here `true` is the default.
Does that mean this is a breaking (behavior is different) change?

Also, the wording here is a little awkward, or maybe wrong?

Reply:
Yeah this was leftover from a previous implementation I was doing that used a different tag with a default value of `true`. Forgot to change that! Good catch

Reply:
Cool.